<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrinterForm</class>
 <widget class="QWidget" name="PrinterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>打印设置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../img.qrc">
    <normaloff>:/img/images/printer.png</normaloff>:/img/images/printer.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;Sans Serif&quot;;</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>231</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>是否开启自动打印功能</string>
   </property>
   <widget class="QRadioButton" name="radiobutton_open">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>118</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开启</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radiobutton_close">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>118</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_close">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>关  闭</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../img.qrc"/>
 </resources>
 <connections/>
</ui>
